### ISSUE TITLE
Use Minima for theme?

### ISSUE THREAD

**Issue by pathawks** [jekyll/benchmarking#4]
If we used Minima as a theme, it might give us a better idea of how a real site would look.

Of course, it would also introduce another variant.

**Comment by DirtyF** [comment]
I made some tests with `minima` and applying layouts makes a real diff like +50% build time with a single layout applied. We need benchmarks with layouts.

Could we run benchmarks against a GitHub theme repo?
Could we isolate variants like in a A/B/n testing?

**Comment by DirtyF** [comment]
Here's a sample benchmark for 1000 posts run with minima, looks like when the date and content doesn't change, generating posts is faster. Should we decrement dates by a day?

```
Filename                                                              | Count |    Bytes |  Time
----------------------------------------------------------------------+-------+----------+------
_layouts/default.html                                                 |  1001 | 7743.29K | 6.840
usr/local/lib/ruby/gems/2.5.0/gems/minima-2.1.1/_includes/head.html   |  1001 |  629.14K | 3.350
usr/local/lib/ruby/gems/2.5.0/gems/minima-2.1.1/_includes/header.html |  1001 | 1229.74K | 1.644
usr/local/lib/ruby/gems/2.5.0/gems/minima-2.1.1/_includes/footer.html |  1001 |  549.38K | 0.917
_layouts/post.html                                                    |  1000 | 4935.44K | 0.744
_layouts/home.html                                                    |     1 |  208.96K | 0.499
_posts/2018-02-06-test_post_1.md                                      |     1 |    3.94K | 0.279
_posts/2018-02-06-test_post_320.md                                    |     1 |    3.94K | 0.006
_posts/2018-02-06-test_post_608.md                                    |     1 |    3.94K | 0.003
_posts/2018-02-06-test_post_502.md                                    |     1 |    3.94K | 0.001
_posts/2018-02-06-test_post_746.md                                    |     1 |    3.94K | 0.001
_posts/2018-02-06-test_post_3.md                                      |     1 |    3.94K | 0.001
_posts/2018-02-06-test_post_421.md                                    |     1 |    3.94K | 0.001
_posts/2018-02-06-test_post_785.md                                    |     1 |    3.94K | 0.001
...
                    done in 16.374 seconds.
```

**Comment by ashmaroli** [comment]
> Should we decrement dates by a day?

Yes, that would be best, because a real site would have posts published on the same day *and* over multiple days.. So, if there's a performance sink somewhere, we need to be aware of it's existence..

**Comment by DirtyF** [comment]
fixed by #8
